src/webgpu/shader/validation/expression/call/builtin/value_constructor.spec.ts

```diff
@@ -76,7 +77,153 @@ g.test('vector_splat').unimplemented();
 
 g.test('vector_copy').unimplemented();
 
-g.test('vector_elementwise').unimplemented();
+g.test('vector_elementwise')
+  .desc('Test element-wise vector constructors')
+  .params(u =>
+    u
+      .combine('type', [
+        'bool',
+        'i32',
+        'u32',
+        'f32',
+        'f16',
+        'abstract-int',
+        'abstract-float',
+      ] as const)
+      .combine('ele_type', [
+        'bool',
+        'i32',
+        'u32',
+        'f32',
+        'f16',
+        'abstract-int',
+        'abstract-float',
```

Comment:
What about checking for things like putting a matrix here, or texture or sampler, etc. The various things that can't be params to make sure they fail?

Reply:
I can add some coverage.